### PR TITLE
Merge Redesign into Master branch

### PR DESCRIPTION
**Pull request by tomdng**
So the redesign branch represents the conversion that I did to Gatsby, so the code will probably look like an entirely new project.

`src/components` will have the original React components from the previous setup minus the fact that TutorialComponent now uses graphlql queries to get tutorial information. `src/pages/index.tsx` will include the really basic global state that I setup.

Aside from that there's just a completely new set of CSS to reflect the redesign.

EDIT: Also Codacy thinks that we shouldn't be using semicolons for some reason even though the linting enforces semicolons... Not sure if Codacybot needs to be adjusted or if I should add in an explicit ESLint rule to require semicolons.

### CODE REVIEW

**Comment by tomdng** [comment]
When I did the review for the other PR I just realized there was stuff I was missing like the 404 page being untouched. I've made a few small fixes that shouldn't affect the main app.

File: src/components/navbar/index.tsx [hunk 1]
```diff
@@ -30,12 +57,20 @@ const Navbar: React.FC = (): JSX.Element => {
       <NavLogo
         to="/"
         onClick={(): void => {
-          setTutorialDisplay(false);
-          setSelectorDisplay(true);
+          setTutorialDisplay(true);
+          setSelectorDisplay(false);
```
Comment (ClayMav) [inline src/components/navbar/index.tsx]: can you move this into a defined function? React has a performance penalty for functions defined inline

File: src/components/navbar/index.tsx [hunk 2]
```diff
@@ -30,12 +57,20 @@ const Navbar: React.FC = (): JSX.Element => {
...
-        Propguard
+        <Logo />
       </NavLogo>
+      <NavIcon
+        icon={listIcon}
+        width="2.5rem"
+        onClick={(): void => {
+          setTutorialDisplay(false);
+          setSelectorDisplay(true);
+        }}
```
Comment (ClayMav) [inline src/components/navbar/index.tsx]: same here

**Review by ClayMav — APPROVED** [review]
This code is great, and it looks amazing on my machine and my ultrawide. Great work to you and Henri.

**Comment by tomdng** [comment]
Awesome thanks for the review and feedback! Glad to see it doesn't look off on wider monitors. I've turned the tutorial/selector toggles into a single defined function.

**Pull request merged**